src/__type_tests__/card-type-test.tsx

```diff
@@ -42,8 +47,14 @@ import Image from '../image';
 <PosterCard title="title" trackingEvent={{name: 'do-something'}} />;
 // @ts-expect-error backgroundImage and backgroundVideo can't be used together
 <PosterCard title="title" backgroundImage="" backgroundVideo="" />;
-// @ts-expect-error backgroundImage or backgroundVideo are mandatory
+// @ts-expect-error backgroundImage and backgroundColor can't be used together
+<PosterCard title="title" backgroundImage="" backgroundColor="" />;
+// @ts-expect-error backgroundVideo and backgroundColor can't be used together
+<PosterCard title="title" backgroundVideo="" backgroundColor="" />;
+// @ts-expect-error backtroundColor, backgroundImage or backgroundVideo are mandatory
 <PosterCard title="title" />;
+// @ts-expect-error if you set a custom backgroundColor, you should specify the variant
```

Comment:
Why not using a default value for the variant if it's not provided?

Reply:
I think I prefer this to be explicit. You set either `backgroundImage`, `backgroundColor` or `backgroundVideo`. `backgroundImage` can be set explicitly with a custom color (`backgroundColor="red"`) or implicitly using an specific variant (`variant="inverse"` or `isInverse`)